File: List View Demo/app/src/main/java/com/hasib/listviewdemo/MainActivity.java
package com.hasib.listviewdemo;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.AdapterView;
import android.widget.ArrayAdapter;
import android.widget.ListView;
import android.widget.Toast;

import java.util.ArrayList;
import java.util.Arrays;
import java.util.List;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ListView listView = (ListView) findViewById(R.id.myListView);

        List<String> list = new ArrayList<>(
                Arrays.asList("Toufiq", "Wadud", "Sunny", "Ashraful"));
        ArrayAdapter arrayAdapter = new ArrayAdapter(this,
                android.R.layout.simple_list_item_1, list);
        listView.setAdapter(arrayAdapter);

        listView.setOnItemClickListener(new AdapterView.OnItemClickListener() {
            @Override
            public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
                Log.i("Name: ", parent.getItemAtPosition(position).toString());

                Toast.makeText(MainActivity.this,
                        parent.getItemAtPosition(position).toString(), Toast.LENGTH_LONG).show();
            }
        });
    }
}

File: _Kotlin-Camouflage/CustomViewElements/app/src/main/java/com/hasib/customviewelements/MainActivity.kt
package com.hasib.customviewelements

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val box = findViewById<ValueBox>(R.id.mainBox)
        box.apply {
            setLabel("X")
            setValue("100000")
            setOnClickListener{
                Toast.makeText(context, "Fucking working", Toast.LENGTH_SHORT).show()
            }
        }
    }
}